test_schemas/en/test_checkbox_instruction.json

```diff
@@ -104,7 +104,7 @@
                                     {
                                         "id": "custom-label-checkbox-answer",
```

Comment:
All of the ids in this schema need to be renamed to use `instruction` rather than `label`.